source/faq.txt

```diff
@@ -17,6 +17,21 @@ This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
 
+Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
+---------------------------------------------------------------------------------------
+
+You can perform reads when ``canCommit`` is ``true``. However, the data that you 
+read is :term:`eventually consistent <eventual consistency>`, meaning that you 
+may not always read the latest writes.
+
+You can write to any non-synced namespaces while sync is in progress. If you 
+write to  a namespace before issuing a :ref:`commit <c2c-api-commit>` and 
+waiting for ``canWrite: true``, the behavior is undefined. You can avoid this 
```

Comment:
[suggestion]

This might be slightly clearer written as:

"... before issuing a commit and while `canWrite` is `false`, the behavior is undefined."